config/default/s3fs.settings.yml

```diff
@@ -0,0 +1,25 @@
+access_key: ''
```

Comment:
Nice, looks like we can configure all of this in settings.php (see https://git.drupalcode.org/project/s3fs/blob/7.x-2.x/README.txt#L224)

Reply:
Yes, that will be possible if required.